optimizely/optimizely_config.py

```diff
@@ -68,6 +77,7 @@ def __init__(self, project_config):
             self.is_valid = False
             return
 
+        self._datafile = project_config.to_datafile()
```

Comment:
This should be `datafile = project_config.to_datafile()`. Doesn't have to be class variable.

Reply:
I think you can pass project_config.to_datafile() directly, otherwise I think it has to be class variable to access  it in class method

Reply:
I see. Then in that case the call to `project_config.to_datafile` can be made in `get_config` itself right?

Reply:
correct, @pthompson127 please address it

Reply:
I thought it was more in line with the surrounding code to only save reference to something derived from `project_config`, rather that save reference to the entire `project_config`.

Reply:
Calling `project_config.to_datafile` in `get_config` requires a reference to `project_config`. Which one should I keep a reference to?: `project_config` or `datafile`?

Reply:
yes, I guess nevermind, we will leave datafile instead of project config, current implementation looks good

Reply:
Current is fine.